packages/cli/src/commands/network/parameters.ts

```diff
@@ -0,0 +1,106 @@
+import { BaseCommand } from '../../base'
+
+export default class Parameters extends BaseCommand {
+  static description = 'View network parameters'
+
+  static flags = {
+    ...BaseCommand.flags,
+  }
+
+  async run() {
```

Comment:
We are doing **many calls** here. We could easily run many in parallell, using something like `Promise.all()`